README.md

```diff
@@ -196,6 +196,48 @@ Options:
 Examples:
   shep logs                 Launch an interactive CLI
   shep logs production foo  Shows logs for the `foo` function in the production environment
+
+```
+#### `shep config list`
+```
+shep config list [env] [functionName]
+
+Options:
+  --version  Show version number                                                                               [boolean]
+  --help     Show help                                                                                         [boolean]
+
+Examples:
+  shep config list            Launches an interactive CLI to choose env and functionName
```

Comment:
@southpolesteve mentioned that too. I think once we adopt shep 4.0 we can try to make it a rule that config is only added/removed from the command line, but i do think this is a feature worth pushing either in 4.0 or immediately after